datasets/beschermdestadsdorpsgezichten/dataset.json

```diff
@@ -58,6 +58,10 @@
             "type": "string",
             "description": "De naam van het beschermde stads- of dorpgezicht"
           },
+          "aangewezenDoor": {
+            "type": "string",
```

Comment:
Je zou nog kunnen overwegen hier een enum te introduceren. Wellicht nice to have voor volgende verise.

Reply:
Ik zal dit comment nu resolven, zodat ik kan mergen.